src/elements/content-sharing/utils.js

```diff
@@ -0,0 +1,148 @@
+// @flow
```

Comment:
A small part of me thinks this one should live with the USM or as a separate feature utility, as other BUIE features may need to use/extend the normalizer. I could imagine there would be one function for each defined API object. Responses that already match could be simple pass-thru methods. ones that are not possible/implemented could return `null` or similar. 

`normalize${Box API Object}Response` is a serviceable name. In a similar case, I might also suggest convert as the prefix. Normalize might imply some DB-type operation like converting redundant data into lists and objects.

Reply:
Good idea. I moved this file into the USM utils folder under the name `convertData`. For now, it's still specific to the ContentSharing API use cases, but we can make it more generic later on if necessary.